packages/util/src/use-close-on-document-events.ts

```diff
@@ -7,12 +7,14 @@ export const onGlobalEventsClose = <El extends HTMLElement>(
   el: El,
   callback: Callback
 ) => {
-  if (!canUseDOM()) return
+  if (!canUseDOM()) return noop
 
   const handleClickOutsideMenu = (evt: MouseEvent) => {
     if (evt.target instanceof HTMLElement) {
       if (el.contains(evt.target)) return
-      callback(evt)
+      ;(function cb(): void {
```

Comment:
what's going on here. nextTick maybe. trying to fix a race condition?

Reply:
Opps, part of my earlier debugging and forgot to revert